kbase-extension/static/kbase/js/widgets/kbasePrompt2.js

```diff
@@ -86,8 +86,8 @@ define([
             // into the DOM, in case you were wondering how that magic happens.
             this.$dialogModal = this.createDialogModal();
             this.$dialogModal.on('hidden.bs.modal', function () {
-                // Zap the model.
-                $(this).html('');
+                // Zap the modal.
+                $(this).remove();
```

Comment:
$.html('') empties the element, but leaves the element itself. In this case, the element had a `.modal` class in it, which was making other unit tests go bonkers when they found it with a querySelector. The intent here seems to be to remove the element all together, so I updated.